normalizer.py

```diff
@@ -0,0 +1,137 @@
+import string
+
+import inflect
+
+
+class Normalizer:
+    AMERICAN_SPELLINGS = {
```

Comment:
Let's move this over to WER calculation and select the best score for increased accuracy.

Reply:
Done!